src/tls13.c

```diff
@@ -850,6 +850,26 @@ static int DeriveTrafficSecret(WOLFSSL* ssl, byte* secret)
                      ssl->specs.mac_algorithm, 0);
 }
 
+
+static inline int Tls13_HKDF_Extract(WOLFSSL *ssl, byte* prk, const byte* salt, int saltLen,
```

Comment:
Why force inline? If so please use our `WC_INLINE`. My preference is let the compiler decide and remove it.